sdks/python/src/opik/opik_configure.py

```diff
@@ -19,6 +17,9 @@
 HEALTH_CHECK_URL_POSTFIX: Final[str] = "/is-alive/ping"
 HEALTH_CHECK_TIMEOUT: Final[float] = 1.0
 
+URL_ACCOUNT_DETAILS: Final[str] = "https://www.comet.com/api/rest/v2/account-details"
```

Comment:
It's better to get base url from `url_override`.
In that case it will work for onprem installations as well(like `https://www.somecompany.comet.com/`).

Reply:
I will change it in next PR as we discussed today.